docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+### Genesis File
+
+The genesis file is a file that contains the initial configuration of the Subnet. Avalanche will generate a genesis file for you based on some parameters you provide. On the other hand, you can also create your own genesis file! This will allow you to have more control over the configuration of your Subnet.
+
+## Creating a Simple Subnet
+
+For this time, let's take the easy way out and use the CLI to deploy a Subnet to see how things work.
+
+```bash
+$ avalanche Subnet create testSubnet
+```
+
+This command will run the wizard to create a Subnet.
+
+There are six steps in the wizard:
+
+#### Choose Your VM
+
+Choose SubnetEVM here
+
+#### Chain Id
+
+Chain id is a unique identifier for the network. This value must be unique so check [chainlist](https://chainlist.org/) to see if the chain id is already in use.
+
+#### Token Symbol
+
+Token symbol is the symbol of the native token used in the Subnet. For instance, on the C-Chain, the symbol is `AVAX` and on the Ethereum mainnet, the symbol is `ETH`. You can use any symbol you want.
+
+#### Gas Configuration
+
+This step will define the gas configuration of your network. You can choose a preset fee configuration or create your own. Keep in mind that more transaction throughput means more disk usage.
+
+We will go with C-Chain default fees.
+
+```bash
+> Low disk use    / Low Throughput    1.5 mil gas/s (C-Chain's setting)
+```
+
+#### Airdropping Native Tokens
+
+This step will define how you want to distribute the funds in your Subnet. You can choose the addresses and amounts you want to distribute.
+Let's go with the simplest case.
+
+```bash
+How would you like to distribute funds?
+> Airdrop 1 million tokens to the default address (do not use in production)
+```
+
+#### Adding a Custom Precompile To Modify the EVM
+
+```bash
+? Advanced: Would you like to add a custom precompile to modify the EVM?: 
+  ▸ No
+    Yes
+    Go back to previous step
+```
+
+Here you can set up a custom precompile to unlock some useful functions.
+
+##### Native Minting
+
+This precompile allows admins to permit designated contracts to mint the native token on your Subnet.
+We will discuss this in more detail in the [`contractNativeMinterConfig`](#contractNativeMinterConfig) section.
+
+##### Configure Contract Deployment Whitelist
+
+This precompile restricts who has the ability to deploy contracts on your Subnet.
+We will discuss this in more detail in the [`contractDeployerAllowListConfig`](#contractdeployerallowlistconfig) section.
+
+##### Configure Transaction Allow List
+
+This precompile restricts who has the ability to make transactions on your Subnet.
+We will discuss this in more detail in the [`txAllowListConfig`](#txAllowListConfig) section.
+
+However, we are not gonna do any of these yet. Choose `No` and proceed.
+
+```bash
+> Successfully created genesis
+```
+
+Congrats! You just created a Subnet! 👏🎉
+## Deploying the Subnet
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Headings should be surrounded by blank lines.

> MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below]

[More info: MD022](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines)